app/views/api/questions/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.array! @questions
```

Comment:
@akshay1y2 Don't need to show all fields, show public fields only. Also show topics